### PR TITLE
Add squad dedicated server template

### PR DESCRIPTION
**Pull request by adrianopteodoro**
Add a template for creation of dedicated server for game [Squad](https://joinsquad.com/)

### CODE REVIEW

File: squad/squad.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "query_port": {
+      "type": "string",
+      "desc": "Steam query port",
+      "display": "Steam Query Port",
+      "required": true,
+      "value": "27165"
+    },
+    "random": {
+      "type": "string",
+      "desc": "Randomize map rotation (ALWAYS, FIRST, NONE)",
```
Comment (LordRalex) [inline squad/squad.json]: We support options for such things, see: https://github.com/PufferPanel/templates/blob/2cd30d6f25b8b20e820527fa8615e4f64638cb78/agario/agario.json#L37

File: squad/squad.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      "display": "Port",
+      "required": true,
+      "value": "7787"
+    },
+    "query_port": {
+      "type": "string",
+      "desc": "Steam query port",
+      "display": "Steam Query Port",
+      "required": true,
+      "value": "27165"
```
Comment (LordRalex) [inline squad/squad.json:68]: Should be a number instead

File: squad/squad.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    },
+    "port": {
+      "type": "string",
+      "desc": "Game server port",
+      "display": "Port",
+      "required": true,
+      "value": "7787"
+    },
+    "query_port": {
+      "type": "string",
```
Comment (LordRalex) [inline squad/squad.json]: Number, not string

File: squad/squad.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      "display": "Fixed Max Players",
+      "required": true,
+      "value": "80"
+    },
+    "port": {
+      "type": "string",
+      "desc": "Game server port",
+      "display": "Port",
+      "required": true,
+      "value": "7787"
```
Comment (LordRalex) [inline squad/squad.json:61]: Number, not string

File: squad/squad.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  "data": {
+    "fixed_max_players": {
+      "type": "string",
+      "desc": "Player count cannot go higher than this",
+      "display": "Fixed Max Players",
+      "required": true,
+      "value": "80"
+    },
+    "port": {
+      "type": "string",
```
Comment (LordRalex) [inline squad/squad.json]: Number, not string

File: squad/squad.json [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+        "commands": ["chmod +x ./SquadGameServer.sh"],
+        "type": "command"
+      }
+    ],
+    "post": [],
+    "environmentVars": {}
+  },
+  "data": {
+    "fixed_max_players": {
+      "type": "string",
```
Comment (LordRalex) [inline squad/squad.json]: Number, not string

File: squad/squad.json [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      "desc": "Player count cannot go higher than this",
+      "display": "Fixed Max Players",
+      "required": true,
+      "value": "80"
```
Comment (LordRalex) [inline squad/squad.json:54]: Number, not string

File: squad/README.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# SQUAD Dedicated Server
+
+## Template OS suport
+
+This template only works for the folling operating systems:
```
Comment (LordRalex) [inline squad/README.md]: Typo: "folling"

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: squad/squad.json (hunk 4 above)
Comment (adrianopteodoro) [reply squad/squad.json:61]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 3 above)
Comment (adrianopteodoro) [reply squad/squad.json]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 2 above)
Comment (adrianopteodoro) [reply squad/squad.json:68]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 5 above)
Comment (adrianopteodoro) [reply squad/squad.json]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 7 above)
Comment (adrianopteodoro) [reply squad/squad.json:54]: Thanks!! The adjustments has been made 👌

File: squad/README.md (hunk 8 above)
Comment (adrianopteodoro) [reply squad/README.md]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 1 above)
Comment (adrianopteodoro) [reply squad/squad.json]: Thanks!! The adjustments has been made 👌

File: squad/squad.json (hunk 6 above)
Comment (adrianopteodoro) [reply squad/squad.json]: Thanks!! The adjustments has been made 👌

**Comment by adrianopteodoro** [comment]
Thanks @LordRalex!! I've made the code adjustments requested 👌

**Pull request merged**